File: Estrutura_Condicional/C/Ex1040/main.c
#include <stdio.h>

int main(){

    float n1, n2, n3, n4, media;
    scanf("%f" "%f" "%f" "%f", &n1, &n2, &n3, &n4);

    media = (n1 * 2 + n2 * 3 + n3 * 4 + n4) / 10;

    printf("Media: %.1f\n", media);

    if (media >= 7) {
        printf("Aluno aprovado.\n");
    } else if (media < 5){
        printf("Aluno reprovado.\n");
    } else if (media >= 5 && media <= 6.9) {
        printf("Aluno em exame.\n");
        float ne;
        scanf("%f", &ne);
        printf("Nota do exame: %.1f\n", ne);
        media = (media + ne) / 2;

        if (media >= 5){
            printf("Aluno aprovado.\nMedia final: %.1f\n", media);
        } else if (media <= 4.9) {
            printf("Aluno reprovado.\nMedia final: %.1f\n");
        }
    } 

    return 0;
}

File: Estrutura_Condicional/C#/Ex1040/Main.cs
using System;

class Program
{
    static void Main(string[] args)
    {
        string[] notas = Console.ReadLine().Split(' ');
        double N1 = double.Parse(notas[0]);
        double N2 = double.Parse(notas[1]);
        double N3 = double.Parse(notas[2]);
        double N4 = double.Parse(notas[3]);

        double media = (N1 * 2 + N2 * 3 + N3 * 4 + N4 * 1) / 10;
        Console.WriteLine($"Media: {media:f1}");

        if (media >= 7.0)
        {
            Console.WriteLine("Aluno aprovado.");
        }
        else if (media < 5.0)
        {
            Console.WriteLine("Aluno reprovado.");
        }
        else
        {
            Console.WriteLine("Aluno em exame.");
            double notaExame = double.Parse(Console.ReadLine());
            Console.WriteLine($"Nota do exame: {notaExame:f1}");

            double mediaFinal = (media + notaExame) / 2;
            Console.WriteLine(mediaFinal >= 5.0 ? "Aluno aprovado." : "Aluno reprovado.");
            Console.WriteLine($"Media final: {mediaFinal:f1}");
        }
    }
}